tests/test_utils.py

```diff
@@ -15,30 +15,50 @@
 """Tests utility functions"""
 
 from semantic_version import Version
-from release_bot.utils import process_version_from_title
+from release_bot.utils import (process_version_from_title,
+                               look_for_version_files)
 
 
 def test_process_version_from_title():
     """Test converting title into version as per SemVer versioning"""
     latest_version = Version("0.0.1")
 
     title = '3.7.8 release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == "3.7.8"
-    assert match == True
+    assert match is True
     title = 'new major release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == "1.0.0"
-    assert match == True
+    assert match is True
     title = 'new minor release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == "0.1.0"
-    assert match == True
+    assert match is True
     title = 'new patch release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == "0.0.2"
-    assert match == True
+    assert match is True
     title = 'random release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == ""
-    assert match == False
+    assert match is False
+
+
+def test_look_for_version_files(tmp_path):
```

Comment:
Wonderful, I haven't known `tmp_path`, I've been always using `Path(str(tmpdir))`

Reply:
Props to pytest for making the new and fancy features easy to lookup and use (...of course I couldn't remember how to do this without googling)